src/components/ha-ansi-to-html.ts

```diff
@@ -34,7 +34,7 @@ export class HaAnsiToHtml extends LitElement {
 
   @litState() private _filter = "";
 
-  protected render(): TemplateResult | void {
+  protected render(): TemplateResult | undefined {
```

Comment:
When does this return undefined?

Reply:
:shrug:  I just replaced void with undefined, I'll remove it.